litmus/AArch64Compile_litmus.ml

```diff
@@ -918,6 +939,336 @@ module Make(V:Constant.S)(C:Config) =
         outputs = [r1];
         reg_env = (add_128 [r1;r2;r3;])}
 
+(* Scalable vector Extension *)
+    let pp_pattern = function
+      | POW2 -> "pow2"
+      | VL1 -> "vl1"
+      | VL2 -> "vl2"
+      | VL3 -> "vl3"
+      | VL4 -> "vl4"
+      | VL5 -> "vl5"
+      | VL6 -> "vl6"
+      | VL7 -> "vl7"
+      | VL8 -> "vl8"
+      | VL16 -> "vl16"
+      | VL32 -> "vl32"
+      | VL64 -> "vl64"
+      | VL128 -> "vl128"
+      | VL256 -> "vl256"
+      | MUL4 -> "mul4"
+      | MUL3 -> "mul3"
+      | ALL -> "all"
+
+    let print_preg io offset i r = match r with
+    | Preg (_,s) -> "^" ^ io ^ string_of_int (i+offset) ^
+       (try Misc.lowercase (List.assoc s sve_arrange_specifier) with Not_found -> assert false)
+    | PMreg (_,s) -> "^" ^ io ^ string_of_int (i+offset) ^
+       (try Misc.lowercase (List.assoc s sve_pred_modifier) with Not_found -> assert false)
+    | _ -> assert false
+
+    let print_zreg io offset i r = match r with
+    | Zreg (_,s) -> "^" ^ io ^ string_of_int (i+offset) ^
+      (try Misc.lowercase (List.assoc s sve_arrange_specifier) with Not_found -> assert false)
+    | _ -> assert false
```

Comment:
It would be more elegant to define `pp_sve_arrange_specifier` and `pp_sve_pred_modifier` functions in `lib/AArch64Base.ml` and then:
```suggestion
    let print_preg io offset i r = match r with
    | Preg (_,s) -> "^" ^ io ^ string_of_int (i+offset) ^
       (pp_sve_arrange_specifier s |> Misc.lowercase)
    | PMreg (_,s) -> "^" ^ io ^ string_of_int (i+offset) ^
       (pp_sve_pred_modifier s |> Misc.lowercase)
    | _ -> assert false

    let print_zreg io offset i r = match r with
    | Zreg (_,s) -> "^" ^ io ^ string_of_int (i+offset) ^
      (pp_sve_arrange_specifier s |> Misc.lowercase)
    | _ -> assert false
```